### PR TITLE
Add licenses

### PR DESCRIPTION
**Pull request by lschmierer**

### CODE REVIEW

File: LICENSE-MIT [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2015 The sacn Developers
```
Comment (ebkalderon) [inline LICENSE-MIT]: Um... I think this should say `Copyright (c) 2016 Amethyst Developers` instead. :wink:

Comment (lschmierer) [inline LICENSE-MIT]: fixed, with nobody ever notices in the git log :P

**Comment by ebkalderon** [comment]
I guess this is fine. But just to clarify, when calling `amethyst new`, shouldn't we *not* be copying in these license files nor the `README.md` as well? I don't like the idea of automatically attributing every engine user's work to "The Amethyst Developers".

**Comment by lschmierer** [comment]
Well, since the engine itself is/will be Apache and MIT, it is required for users to include the license files anyway.

**Comment by ebkalderon** [comment]
I don't think we need to include either in the resulting `amethyst new` project. The idea is that people can apply whatever license they want to their work, including proprietary licenses.

Besides, when I link to `toml-rs` in any project, for example, I'm not required to include the MIT/Apache license disclaimers with it.

**Pull request closed without merging**